GameLogic/engine/StandardFENSerializer.cs

```diff
@@ -41,6 +41,16 @@ public static class StandardFENSerializer
                 {'b', Color.Black}
             };
 
+        public static Dictionary<char, PieceType> mappingNotationToPieceType = new Dictionary<char, PieceType>()
+            {
+                {'N', PieceType.Knight},
```

Comment:
Just in case, in the UCI protocol in case of the pawn promotion the piece type is written in lowercase.

http://wbec-ridderkerk.nl/html/UCIProtocol.html (Search for "Move format")

But I do not think we have to necessarily follow the format from the UCI protocol as in other protocols the format may be different.

Reply:
Fixed!